### ISSUE TITLE
Uses deprecated logger.warn (instead of logger.warning)

### ISSUE THREAD

**Issue by dfrankow** [jjmontesl/cubetl#3]
I'm using python 3.7.4.

This would be pretty easy for me to fix if you wished.

Example:

```
  /app/cubetl/sql/sql.py:372: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead
```

All the lines:
```
$ git grep 'logger.warn('
cubetl/core/components.py:            logger.warn("Finalized a non initialized component: %s" % comp)
cubetl/core/components.py:                logger.warn("Unfinalized component %s" % comp_desc.comp)
cubetl/fs/__init__.py:            logger.warn("Could not stat file: %s", path)
cubetl/http/__init__.py:                logger.warn("Could not retrieve HTTP document (attempt %d/%d): %s " % (attempt_count, self.attempts, e))
cubetl/olap/sql.py:            logger.warn("%s has multiple primary keys mapped: %s (ignoring)" % (self, pk_mappings))
cubetl/olap/sql.py:                            logger.warn("%s looked up an entity which exists with different attributes (field=%s, existing_value=%r, tried_value=%r) (reported only once per field)" % (self, mapping.sqlcolumn, v1, v2))
cubetl/olap/sqlschema.py:                        logger.warn("Ignoring foreign key reference to self: %s", dbcol.name)
cubetl/olap/sqlschema.py:                        logger.warn("Ignoring foreign key reference from %s.%s to not available entity: %s", dbcol.sqltable.name, dbcol.name, related_fact_name)
cubetl/olap/sqlschema.py:                logger.warn("Multiple primary key found in table %s (not supported, ignoring table)", sqltable.name)
cubetl/olap/sqlschema.py:                logger.warn("No primary key found in table %s (not supported, ignoring table)", sqltable.name)
cubetl/pcaxis/__init__.py:                logger.warn("PCAxisIterator could not parse value: %r (cell: %s)", value, container)
cubetl/sql/schemaimport.py:                        logger.warn("Skipped foreign key %s in table %s, as foreign key column (%s.%s) was not found.", dbcol.name, dbtable.name, list(dbcol.foreign_keys)[0].column.table.name, list(dbcol.foreign_keys)[0].column.name)
cubetl/sql/sql.py:                                logger.warn("%s updating an entity that exists with different attributes, overwriting (field=%s, existing_value=%s, tried_value=%s)" % (self, c.name, v1, v2))
cubetl/sql/sql.py:                        logger.warn("Unicode column %r received non-unicode string: %r " % (column.name, row[column.name]))
cubetl/text/__init__.py:                    logger.warn("Failed to match regular expresion %s on value: %s", self.regexp, data)
```

**Comment by jjmontesl** [comment]
Please, go ahead.

I'd be grateful if you can provide this in a separate pull request (or at least one that contains logging related issues only).

**Comment by jjmontesl** [comment]
I'm closing this as it's been addressed by PR #7.
